WinSCP/winscp.py

```diff
@@ -282,6 +282,11 @@ def _autodetect_official_installreg(self):
             pass
         return None
 
+    def _autodetect_official_installreg(self):
+        hklm = self._exe_from_reg(winreg.HKEY_LOCAL_MACHINE)
+        return (hklm if hklm is not None
+                else self._exe_from_reg(winreg.HKEY_CURRENT_USER))

```

Comment:
Shouldn't this have precedence over HKLM?

Reply:
Sorry, I was away for a while.

I guess you're right, the installation closest to the user should have more precedence. I'll swap the values and test tonight before pushing the change.